### DISCUSSION TITLE
How to render a page in another page directly?

### DISCUSSION THREAD

**Discussion by zw963** [luckyframework/lucky#1797] [Q&A] [answered]
Following is a render method example come from kemal:

https://github.com/one-in-ten-thousand/production_line_report/blob/master/src/views/admin/companies/show.ecr#L3

For the above example, i want to render the `manufactories/index_page.cr` page in `companies/show.cr` directly, but `manufactories/index_page.cr` still serve as manufactories RESTFul index.

I read almost all document for page/action, no clue for this, it seem like we have to wrap the common page into a Component, then
mount it in the page, right?

Thanks.

**Comment by matthewmcgarvey** [comment, marked as answer]
You would extract the parts from `manufactories/index_page.cr` you want to be shown in `companies/show.cr` into a [Component](https://luckyframework.org/guides/frontend/rendering-html#creating-and-using-components) and use it in both pages
